ll.tm

```diff
@@ -4162,7 +4169,15 @@ LinkageNameField -> LinkageNameField
 ;
 
 LowerBoundField -> LowerBoundField
-	: 'lowerBound:' LowerBound=IntLit
+	: 'lowerBound:' LowerBound=MDFieldOrInt
+;
+
+UpperBoundField -> UpperBoundField
```

Comment:
I understand the intention to keep these field definitions close to one another. As some field definitions (e.g. `LineField`) are shared between multiple specialized metadata node definitions, the list of field definitions are sorted alphabetically. So, for consistency, please insert `UpperBoundField` and `StrideField` in alphabetical order below.

Reply:
I have a question is any tool can automatically find such position?

Reply:
> I have a question is any tool can automatically find such position?

For sorting the [token list](https://github.com/llir/grammar/pull/8#discussion_r524576906) I simply use the ["Sort Lines (numerically)"](https://github.com/alimony/sublime-sort-numerically) plugin for Sublime Text.

As for finding the insertion point for these fields, I do it manually. The reason is mainly since new metadata fields are added rather rarely, and as such a manual method seem sufficient.

Cheers,
Robin